### ISSUE TITLE
chat issues with -E (environment variable substitution)

### ISSUE THREAD

**Issue by Giulio-xyz** [ppp-project/ppp#294]
Sometimes, using "-E" won't substitute variables.

To solve the issue, could you consider this:

```

--- ppp/chat/chat.c.orig        2005-11-26 19:30:31.000000000 +0100
+++ ppp/chat/chat.c     2005-11-26 19:30:46.000000000 +0100
@@ -1129,7 +1129,7 @@
        report_string[n_reports++] = s1;
 
        if (verbose)
-           msgf("report (%v)", s);
+           msgf("report (%v)", s1);
        return;
     }
```

For details, see  https://bugzilla.redhat.com/show_bug.cgi?id=1650539

Thanks

**Comment by Neustradamus** [comment]
@paulusmack: What do you think?

**Comment by paulusmack** [comment]
Fixed in commit 9c7ba0d42dee.
